terraform/ec2/win/main.tf

```diff
@@ -152,15 +153,23 @@ resource "null_resource" "integration_test_run" {
   ]
 
   connection {
-    type     = "winrm"
-    user     = "Administrator"
-    password = rsadecrypt(aws_instance.cwagent.password_data, local.private_key_content)
-    host     = aws_instance.cwagent.public_dns
+    type            = "ssh"
+    user            = "Administrator"
+    password        = rsadecrypt(aws_instance.cwagent.password_data, local.private_key_content)
+    host            = aws_instance.cwagent.public_ip
+    target_platform = "windows"
+    timeout         = "6m"
   }
 
   provisioner "remote-exec" {
     inline = [
-      "validator.exe --test-name=${var.test_dir}",
+      "set AWS_REGION=${var.region}",
+      # "validator.exe --test-name=${var.test_dir}",
```

Comment:
nit: I'm guessing this is leftover comment?

Reply:
yeah leaving it here as a future reference